### PR TITLE
Pluralise 'ministers' only where appropriate

### LINKED ISSUE #487: Update to our ministers module on department landing pages
Expected outcome
- When only one minister is tagged in the 'Our ministers' module of a department landing page, the module title needs to change to 'Our Minister'. 

Actual outcome
- Current example of one minister sitting under a pluralised module title: https://gov-au-beta.apps.platform.digital.gov.au/department-of-agriculture-and-water-resources 

@streetdaddy Could you please add anything I've missed?

### PR DESCRIPTION
**Pull request by maxious**
Fixes #487

### CODE REVIEW

File: app/views/templates/_related_ministers.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 - if related_ministers.present?
-  %h2 Our ministers
+  %h2= 'Our ' + (related_ministers.count == 1 ? 'minister': 'ministers')
```
Comment (coderdan) [inline app/views/templates/_related_ministers.html.haml]: You might want to use the pluralize helper.

Comment (maxious) [inline app/views/templates/_related_ministers.html.haml]: That's what I thought but the pluralize helper prefixes the string with the number of things ie. "Our 1 minister", "Our 2 ministers". Looking inside the helper implementation, one could use the .pluralize string method but overkill for this case.

Comment (coderdan) [inline app/views/templates/_related_ministers.html.haml]: Oh true. Yeah thats annoying!

**Comment by coderdan** [comment]
LGTM

**Comment by ltankey** [comment]
LGTM

**Pull request merged**